Mesh_3/doc/Mesh_3/Mesh_3.txt

```diff
@@ -759,9 +759,9 @@ protection of the 1D-features.
 <center>
 <table border=0>
 <tr>
-  <td><img border="0" height="435px" src="./no-protection.png"></td>
-  <td><img border="0" height="435px" src="./protection-box.png"></td>
-  <td><img border="0" height="435px" src="./protection-all.png"></td>
+  <td><img border="0" src="./no-protection.png"></td>
+  <td><img border="0" src="./protection-box.png"></td>
+  <td><img border="0" src="./protection-all.png"></td>
```

Comment:
Why? I have added that parameter on purpose. I think it was Andreas @afabri that suggested to include sizes in the HTML, to avoid that the layout of the documentation page changes during the loading of images.

Andreas, can you confirm?